scripts/register.js

```diff
@@ -2,35 +2,38 @@ const { REST } = require("@discordjs/rest");
 const { Routes } = require("discord-api-types/v9");
 
 const { token, application_id } = require("../config.json");
-
+//the second argument passed is supposed to be the guild id
 const guild = process.argv[2];
 
 const commands = [
-	{
-		name: "snipe",
-		description: "Shows the last deleted message from a specified channel!",
-	},
+  {
+    name: "snipe",
+    description: "Shows the last deleted message from the server!",
+  },
+  {
+    name: "esnipe",
+    description: "Shows the last edited message from the server!",
+  },
 ];
 
 const rest = new REST({ version: "9" }).setToken(token);
 
 (async () => {
-	try {
-		console.log("[sniper] :: Started refreshing application (/) commands.");
+  try {
+    console.log("[sniper] :: Started refreshing application (/) commands.");
 
-		await rest.put(
-			guild
-				? Routes.applicationGuildCommands(application_id, guild)
-				: Routes.applicationCommands(application_id),
-			{
-				body: commands,
-			}
-		);
+    await rest.put(
+      //if there is the guild id then register guild specific sommands otherwise register a global command.
```

Comment:
Not sure if this comment is needed here, this is pretty self explanatory.

Reply:
alright it's there for when you are a newbie [ who not really familiar with the new es6 and stuff ]  and are just looking at code